internal/connector/test/integration/features/connector-admin-api.feature

```diff
@@ -174,3 +174,124 @@ Feature: connector admin api
 
     And UNLOCK--------------------------------------------------------------
 
+  Scenario: Ricky Bobby can stop and start a connector
+    Given LOCK--------------------------------------------------------------
```

Comment:
We shouldn't be using `LOCK/UNLOCK` for tests that should be independent of other tests. Try using new users and orgs to isolate tests from each other. Or, move the steps into another existing scenario that tests related APIs. I know that creates longer scenarios, but if they require a similar env setup, then they belong in the same scenario.